metadata.rb

```diff
@@ -6,4 +6,5 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '0.2.0'
 
-depends 'ark'
+depends 'ark', "= 0.9.0"
```

Comment:
This was troubleshooting step when ark wouldn't install_with_make, accidentally left it in, removing now...